CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 3.0.0 (to be released)
+
+* [BC] removed deprecated features
```

Comment:
should be `[BC break]`, not `[BC]` (the part making it worth highlighting it is precisely `break`)